GoTeamApp/GoTeamApp/EditLabelViewController.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+class EditLabelViewController: UIViewController {
+    
+    @IBOutlet weak var labelNameField: UITextField!
+    var label: Labels?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        labelNameField.text = label?.labelName!
```

Comment:
FYI, using  labelNameField.text = label?.labelName here would have worked too, since 'text' is an optional. I personally, generally avoid using forced unwrapping as much as possible since this leads to crashes.